designs/virtualization/4.x/cd roms/cd roms.md

```diff
@@ -0,0 +1,39 @@
+---
+parent: Virtualization
+version: 4.x
+---
+# CD ROMs
+
+## CD-ROMs on VM Details page
+
+It's been determined that separating CD-ROMs from the rest of the disks has technical overhead and complicates the flow. It makes more sense to include them as a lower level item in the disks rather than highlight them in the details view like we currently do.
+
+![CDROMs are on the details page](img/Details.png)
+
+We'll add a ‘Drive’ column to the ‘Disks’ table view.
+The default type will be ‘Disk’ and the other option(s) will be ‘CD ROM’.
+
+![adding a new Drive column](img/Type-column-added-table-view.png)
+
+## CD-ROMs on Create VM wizard
+
+We'll remove the CD ROMs from the ‘Advanced’ step and place it in the Storage step.
+For cases where we point the user to them (windows guest tools), we’ll direct the users via a notification with a link to the Disks table under the Storage step.
+
+![notification directing to the disks table](img/Notification-to-disks-table.png)
```

Comment:
I don't think this would be an inline notification - it would still stay as help text with the input right?

Reply:
@matthewcarleton in which input field? the host?

Reply:
+1